src/dotnet/commands/dotnet-pack/SymbolPackageGenerator.cs

```diff
@@ -26,8 +26,10 @@ protected override void ProcessContext(ProjectContext context)
             base.ProcessContext(context);
 
             var inputFolder = ArtifactPathsCalculator.InputPathForContext(context);
-            TryAddOutputFile(context, inputFolder, $"{Project.Name}.pdb");
-            TryAddOutputFile(context, inputFolder, $"{Project.Name}.mdb");
+            var ouptutName =  Project.GetCompilerOptions(context.TargetFramework, Configuration).OutputName;
```

Comment:
Which part are you asking about? `GetCompilerOptions`? I don't think it can be ever null and I did not see any code that guards against null. If you are asking about `OutputName` - it falls back to `Project.Name` if `outputName` was not specified (https://github.com/dotnet/cli/blob/rel/1.0.0/src/Microsoft.DotNet.ProjectModel/Project.cs#L116-L119) and again I don't think it can ever be null and we never check if it can be null.